ros/humble/ubuntu/jammy/ros-core/Dockerfile

```diff
@@ -32,6 +32,7 @@ RUN echo "deb [ signed-by=/usr/share/keyrings/ros2-latest-archive-keyring.gpg ]
 ENV LANG C.UTF-8
 ENV LC_ALL C.UTF-8
 
+ENV ROS_SYNC_DATE=2024-09-19
```

Comment:
Should the name of the env var include something which clarifies it's the repository sync date?

`ROS_SYNC_DATE` seems a bit ambiguous.

Reply:
@gavanderhoorn thabnks for the feedback, I was actually looking for feedback on that.

In my mind everytime we speak about "ROS SYNC" it's the sync of the apt repo but its indeed not super explicit.

What do you have in mind ?
something like `ROS_APT_SYNC_DATE` ?

Reply:
> In my mind everytime we speak about "ROS SYNC" it's the sync of the apt repo

same for me, but that's probably because we're intimately familiar with the ROS release workflow/pipeline.

> something like `ROS_APT_SYNC_DATE` ?

that's already more specific, but perhaps `ROS_PKG_REPOS_SYNC_DATE` would be more self-explanatory? It would also work for non-apt based systems and "package repositories" seems like it would be somewhat intuitively understandable.

Reply:
This was discussed in ROS Infrastructure PMC today. summary and suggestion available here https://github.com/osrf/docker_templates/pull/110/files#r1847080537

Reply:
We went with `ROSDISTRO_PKGS_SYNC_DATE`